### PR TITLE
Issue #256: Remove per-user theme support.

### PR DESCRIPTION
**Pull request by jenlampton**
PR for https://github.com/backdrop/backdrop-issues/issues/256

### CODE REVIEW

File: core/modules/user/user.install [hunk 1]
```diff
@@ -335,5 +328,12 @@ function user_update_1001() {
 }
 
 /**
+ * Remove the ability for users to select a theme.
+ */
+function user_update_1002() {
+  db_query("ALTER TABLE {users} DROP theme");
```
Comment (quicksketch) [inline core/modules/user/user.install]: This should use db_drop_field('users', 'theme') instead.

**Comment by jenlampton** [comment]
cleaned up. also rebased.

**Comment by quicksketch** [comment]
Looks great! Merged in. Thanks!

**Comment by quicksketch** [comment]
Looks like I didn't properly merge this the first time around. Now it's merged in for real. Thanks again!

**Pull request merged**